Danish/install.php

```diff
@@ -4,43 +4,44 @@
 
 $lang_install = array(
 
-'Install message'				=>	'The file \'config.php\' doesn\'t exist or is corrupt. This implies that FluxBB has not yet been installed.',
-'Choose install language'		=>	'Choose the install script language',
-'Choose install language info'	=>	'The language used for this install script. The default language used for the board itself can be set below.',
+'Install message'				=>	'Filen \'config.php\' eksistere ikke eller er beskadiget. Dette forudsætter dog at FluxBB endnu ikke er blevet installeret.',
+'Choose install language'		=>	'Vælg sprog til installationen',
+'Choose install language info'	=>	'
+Sproget der anvendes for denne installation. Standard sproget for forummet selv kan vælges nedenfor.',
```

Comment:
There's a line break here at the beginning of the string. Please remove that. :)